### DISCUSSION TITLE
Well-done!

### DISCUSSION THREAD

**Discussion by lloydbayley** [sopelj/hass-ember-mug-component#51] [General]
Just thought I'd say well-done on the latest release. The ability to add/alter presets is excellent! :)

**Comment by sopelj** [comment]
Thanks! Much appreciated :)
I'm glad it's useful. Don't hesitate if you have any suggestions or issues.

**Comment by BillyFKidney** [comment]
Just installed v1.0.0 on HA 2024.2.5 connecting through ESP-Home BlueTooth proxies (multiple in the house).

My only hiccup was due to a change in iOS, the Bluetooth icon in Control Center does not TURN OFF Bluetooth instead it prevents NEW CONNECTIONS for 24 hours. I removed the mug from iOS -> Settings -> Bluetooth and _instantly_ my mug appeared in Home Assistant.
![image](https://github.com/sopelj/hass-ember-mug-component/assets/65253277/10743369-4764-460e-807b-4325e0857aad)


Do you have a [Buy Me a Coffee](https://www.buymeacoffee.com) account? 💲☕️ #SeeWhatIDidThere
